A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -95,11 +95,11 @@ class CDUInitPage {
                     }
                 );
 
-                cruiseFlTemp = "_____|____[color]amber";
+                cruiseFlTemp = "_____\xa0|____[color]amber";
                 //This is done so pilot enters a FL first, rather than using the computed one
                 if (mcdu._cruiseEntered && mcdu._cruiseFlightLevel) {
                     cruiseFlTemp =
-                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "/" +
+                        "{cyan}FL" + mcdu._cruiseFlightLevel.toFixed(0).padStart(3, "0") + "\xa0/" +
```

Comment:
While you're at it, the slash should be small if TEMP is automatic:

![image](https://user-images.githubusercontent.com/929769/111835883-d5c8be00-88f5-11eb-9632-b7de6ea581a6.png)

![image](https://user-images.githubusercontent.com/929769/111835899-dc573580-88f5-11eb-8c77-6aad4443e54d.png)